packages/main.ts

```diff
@@ -10,6 +10,11 @@
 *
 */
 
+import { PerfTiming } from "@zowe/perf-timing";
+
+const timingApi = PerfTiming.api;
+
+timingApi.mark("PRE-INIT");
```

Comment:
Not a big deal, but I wonder if we could  change the name as "PRE-INIT" vs. "BEFORE_INIT" is a little confusing to me. Maybe "PRE_IMPORT_IMPERATIVE" and "POST_IMPORT_IMPERATIVE" or something like that?

Also even if we don't change the name, I think you have a hyphen here but an underscore in the call to .measure below so the measurement might not work without an update

Reply:
I am okay with any change.  The only thing I wanted was all uppercase so they stood out.

Reply:
I'm so sorry I missed this 😢